src/layouts/GlossaryTerms/termsList.module.css

```diff
@@ -1,6 +1,6 @@
 .list {
   margin-top: 3rem;
```

Comment:
I think this is pushing the content down too much since positioning is being done with the grid. Can you try removing?